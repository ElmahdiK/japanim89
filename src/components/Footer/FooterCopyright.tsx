export default function FooterCopyright() {
  return <span className='p-4 text-gray-400'>Â© 2024 japanim89 ğŸ‡¯ğŸ‡µ</span>;
}
