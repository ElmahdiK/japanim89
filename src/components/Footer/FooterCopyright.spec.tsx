import { render, screen } from '@testing-library/react';
import FooterCopyright from './FooterCopyright';

describe('test button theme mode', () => {
  it('renders the footer copyright unchanged', () => {
    const { container } = render(<FooterCopyright />);
    expect(container).toMatchSnapshot();
  });

  it('displays the copyright text', () => {
    render(<FooterCopyright />);
    expect(screen.getByText('Â© 2024 japanim89')).toBeInTheDocument();

    // fireEvent.click(screen.getByText('Load Greeting'));
    // await screen.findByRole('heading');

    // expect(screen.getByRole('heading')).toHaveTextContent('hello there');
    // expect(screen.getByRole('button')).toBeDisabled();
  });
});
