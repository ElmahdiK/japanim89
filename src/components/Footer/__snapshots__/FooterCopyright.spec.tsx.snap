// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button theme mode renders the footer copyright unchanged 1`] = `
<div>
  <a
    class="link p-2 "
    href="./"
  >
    Â© 2024 japanim89
  </a>
</div>
`;
